runtime/queries/log/highlights.scm

```diff
@@ -0,0 +1,10 @@
+(trace) @ui.virtual.inlay-hint
```

Comment:
I think highlight queries should avoid using UI scopes. Could this just be `@comment` instead?